xbmc/video/tags/VideoTagLoaderFFmpeg.cpp

```diff
@@ -118,6 +118,13 @@ bool CVideoTagLoaderFFmpeg::HasInfo() const
   AVDictionaryEntry* avtag=nullptr;
   while ((avtag = av_dict_get(m_fctx->metadata, "", avtag, AV_DICT_IGNORE_SUFFIX)))
     std::cout << avtag->key << ": " << avtag->value << std::endl;
+  std::cout << "---------" << std::endl;
+  for (size_t i = 0; i < m_fctx->nb_streams; ++i)
+  {
+    AVDictionaryEntry* avtag=nullptr;
+    while ((avtag = av_dict_get(m_fctx->streams[i]->metadata, "", avtag, AV_DICT_IGNORE_SUFFIX)))
+      std::cout << avtag->key << ": " << avtag->value << std::endl;
```

Comment:
`std::cout` should be gone, right?

Reply:
debugging code that i keep for the time being. not meant for merge.